src/main/java/org/ibp/api/java/impl/middleware/inventory/InventoryServiceImpl.java

```diff
@@ -38,6 +43,11 @@
 	@Autowired
 	private LocationDataManager locationDataManager;
 
+	private LotIDGenerator lotIDGenerator;
```

Comment:
@vmaletta must be autowired

Reply:
@vmaletta any advantage of having a generator class instead of handling it in the LotService or DataManager?

Reply:
Autowired didn't work. 
Based on ObservationUnitIdGenerator, I've created separate class